master-docker-nonstandard/master.cfg

```diff
@@ -1594,6 +1602,23 @@ c["builders"].append(
     )
 )
 
+c["builders"].append(
+    util.BuilderConfig(
+        name="amd64-ubuntu-2404-debug",
+        workernames=workers["x64-bbw-ubuntu-2404-debug"],
```

Comment:
This should be workers['x64-bbw-docker-ubuntu-2404-debug']
And the list contains the following actual workers:

![image](https://github.com/MariaDB/buildbot/assets/45736827/a5dc008b-03e4-4b34-a916-4034cee4bc06)